src/components/Status/Status.props.ts

```diff
@@ -0,0 +1,12 @@
+import { StatusConfig } from './Status.styles';
+import { StatusAppearance } from './StatusAppearance.type';
```

Comment:
when you're importing types you can use `import type` keyword that way, only types are being imported https://www.typescriptlang.org/docs/handbook/modules/reference.html#type-only-imports-and-exports

```suggestion
import type { StatusAppearance } from './StatusAppearance.type';
```

Reply:
There should be `eslint` error there 🤔

Reply:
A warning `eslint` gives:
<img width="943" alt="Screenshot 2024-04-23 at 09 23 20" src="https://github.com/VirtusLab/tetrisly-react/assets/74671633/cc14d90f-5b9d-49ce-9776-5f1634722a27">
Wouldn't be better to describe exact versions instead of minimal ones '"typescript": "^5.1.6"'. In my last project, @tpiechaczek told us to declare exact versions so as not to use different ones as a result.

Reply:
changed